c7n/offhours.py

```diff
@@ -277,10 +294,41 @@ def get_local_tz(self, parts):
             self.log.warning(
                 "filter:offhours unknown tz %s for %s" % (
                     tz_spec, parts))
-                    
+
             return None
         return tz
-    
+
+
+    def is_custom(self, parts):
+        if len(parts) == 3 and parts.get("on") and parts.get("off"):
```

Comment:
Maybe I should have picked another name for this. So in this case `parts` isn't the same "parts" which passed into `process_current_time` it is the results from the parser. in `is_custom` i figured it was worth having a check to ensure if we want to process custom or default offhours behavior.